.Lib9c.Tests/Model/State/WeeklyArenaStateTest.cs

```diff
@@ -144,7 +149,18 @@ public void GetArenaInfosByFirstRankAndCount(int infoCount, int firstRank, int c
             }
 
             var arenaInfos = weeklyArenaState.GetArenaInfos(firstRank, count);
-            Assert.Equal(expected, arenaInfos.Count);
+            Assert.Equal(expectedCount, arenaInfos.Count);
+
+            if (!arenaInfos.Any())
```

Comment:
is this correct changes?

Reply:
Looking back, it's an unnecessary conditional statement.